fontbe/src/features/kern.rs

```diff
@@ -585,7 +619,8 @@ fn debug_ordered_lookups(
 
 /// All the state needed for splitting kern pairs by script & writing direction
 struct KernSplitContext {
-    gdef: Option<HashMap<GlyphId, GlyphClassDef>>,
+    /// map of all mark glyphs; bool indicates if mark is spacing
```

Comment:
the MarkSpacing enum is not actually a "bool" though, is it? Should it be? maybe you changed it to enum and the comment is stale

Reply:
your deduction is correct 🕵️